alpine-base-s6-python3/Dockerfile.tmpl

```diff
@@ -23,4 +23,5 @@ RUN apk add --update-cache \
     rm -r /usr/lib/python*/ensurepip && \
     pip3 install --upgrade pip setuptools && \
     rm -r /root/.cache && \
-    rm -rf /var/cache/apk/*
+    rm -rf /var/cache/apk/* && \
+    rm -rf /tmp/*
```

Comment:
This is because `python -m ensurepip` leaves a pem file full of root cas in the `/tmp` folder.